contracts/adapters/axelar/AxelarSenderAdapter.sol

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../../interfaces/IBridgeSenderAdapter.sol";
+import "./interfaces/IAxelarChainRegistry.sol";
+import "./interfaces/IAxelarGateway.sol";
+import "./interfaces/IAxelarGasService.sol";
+import "./libs/StringAddressConversion.sol";
+
+interface IMultiBridgeSender {
+    function caller() external view returns (address);
+}
+
+contract AxelarSenderAdapter is Ownable, IBridgeSenderAdapter {
+    IAxelarChainRegistry public axelarChainRegistry;
+    IAxelarGateway public gateway;
+    IAxelarGasService public gasService;
+
+    // A boolean flat indicating whether the contract has been initialized.
+    bool public initialized;
+
+    // default gas limit, can be changed by owner.
+    uint32 public gasLimit = 500000;
+
+    // receiver adapter address on dst chain
+    mapping(uint256 => address) public receiverAdapters;
+
+    // message nonce
+    uint32 public nonce;
+
+    // multiBridgeSender address
+    IMultiBridgeSender public multiBridgeSender;
```

Comment:
This can be removed. The senderAdapter does not check where the message comes from but only relays it.